src/Components/v2/ImageLink.tsx

```diff
@@ -0,0 +1,117 @@
+import { color, Flex, Serif, SerifSize } from "@artsy/palette"
+import React from "react"
+import styled from "styled-components"
+import { space, SpaceProps } from "styled-system"
+import { Media } from "Utils/Responsive"
+
+export const ImageLink: React.SFC<ImageLinkProps> = ({
+  href,
+  imageUrl,
+  children,
+  fontSize,
+  ...rest
+}) => (
+  <LinkWithBackground href={href} imageUrl={imageUrl} {...rest}>
+    <Flex justifyContent="center" alignItems="center" position="relative">
+      <LinkText size={fontSize} textAlign="center" p={1}>
+        {children}
+      </LinkText>
+      <PlaceholderImagesForContainerHeightAndWidth />
+    </Flex>
+  </LinkWithBackground>
+)
+
+interface ImageLinkProps extends SpaceProps, LinkWithBackgroundProps {
+  href: string
+  fontSize?: SerifSize
+}
+
+ImageLink.defaultProps = {
+  fontSize: "5t",
+}
+
+const LinkText = styled(Serif)`
+  position: absolute;
+  vertical-align: middle;
+  text-align: center;
+`
+
+// These images give the container the correct height/width/aspect ratio at each breakpoint.
+//   They are hidden by CSS, but they allow the background image to scale correctly
+//   based on the available width.
+const PlaceholderImagesForContainerHeightAndWidth = () => {
+  return (
+    <div>
+      <Media at="xs">
+        <ImagePlaceholder
```

Comment:
I'm not super happy that these are 3 specific images with specific sizes. It makes this component likely unusable by anyone with different needs. I'm going to look into using SVG's to render these placeholders instead.

Reply:
I made SVG's work!